timescaledb/tutorials/analyze-intraday-stocks/fetch-and-ingest.md

```diff
@@ -0,0 +1,225 @@
+ amount, you need to slice up your requests by month. For example, `year1month1`, 
+ `year1month2`, and so on. Keep in mind that each request can only fetch data for one 
+ symbol at a time.
+
+Here's an example API endpoint:
+
+```
+https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY_EXTENDED&symbol=IBM&interval=1min&slice=year1month1&apikey=your_apikey
+```
+
+Check out the [Alpha Vantage API](https://www.alphavantage.co/documentation/) docs for more information.
+
+### Create the function
+
+Let's start by creating a function that fetches data for one symbol and one month. The function takes these two values as parameters:
+
+* symbol: the ticker symbol you want to fetch data for (e.g. "AMZN" for Amazon).
+* month: an integer value between 1-24 indicating which month you want to fetch data from.
+
+```python
+import config
+def fetch_stock_data(symbol, month):
+    """Fetches historical intraday data for one ticker symbol (1-min interval)
+
+    Args:
+        symbol (string): ticker symbol
+
+    Returns:
+        candlestick data (list of tuples)
+    """
+    interval = '1min'
+
+    # the API requires you to slice up your requests (per month)
+    # like "year1month1", "year1month2", ..., "year2month1" etc...
+    slice = "year1month" + str(month) if month <= 12 else "year2month1" + str(month)
+
+    apikey = config.APIKEY
+
+    # formulate the correct API endpoint with symbol, slice, interval and apikey
+    CSV_URL = 'https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY_EXTENDED&' \
+              'symbol={symbol}&interval={interval}&slice={slice}&apikey={apikey}' \
+              .format(symbol=symbol, slice=slice, interval=interval,apikey=apikey)
+    
+    # read CSV file directly into a pandas dataframe
+    df = pd.read_csv(CSV_URL)
+
+    # add a new symbol column to the dataframe
+    # this is needed as the API doesn't return the symbol value
+    df['symbol'] = symbol
+
+    # convert the time column to datetime object
+    # this is needed so we can seamlessly insert the data into the database later
+    df['time'] = pd.to_datetime(df['time'], format='%Y-%m-%d %H:%M:%S')
+
+    # rename columns to match database schema
+    df = df.rename(columns={'time': 'time', 
+                            'open': 'price_open', 
+                            'close': 'price_close', 
+                            'high': 'price_high',
+                            'low': 'price_low',
+                            'volume': 'trading_volume'})
+
+    # convert the dataframe into a list of tuples ready to be inserted into the database
+    return [row for row in df.itertuples(index=False, name=None)] 
+```
+
+## Insert data into TimescaleDB
+
+When you have the `fetch_stock_data` function working, and you can fetch the candlestick from the API, you can insert it into the database.
+
+To make the ingestion faster, use [pgcopy](https://pgcopy.readthedocs.io/en/latest/) instead of inserting
+data row by row.
```

Comment:
Add to the end of this line:

"TimescaleDB is packaged as an extension to PostgreSQL, meaning all the PostgreSQL tools you know and love already work with TimescaleDB."

Reply:
https://github.com/timescale/docs/pull/150/commits/6f26ecaec1da60757fc45c4c2dba2dd749700b25